quickstart/clean.sh

```diff
@@ -8,46 +8,50 @@ echo "Cleaning..."
 
 # Participant 1: Fluid (OpenFOAM)
 Participant1="Fluid"
-cd ${Participant1}
+cd fluid-openfoam
     # Clean the case
     cleanCase
```

Comment:
Let's add a similar message as with the clean case here. Reason: the `cleanCase` script may not be available (e.g. the user forgot to load the OpenFOAM environment).

```suggestion
    echo "Cleaning the fluid-openfoam case"
    cleanCase
```